### PR TITLE
Expanded allow list to include Pytorch Dev Infra

### PR DESCRIPTION
**Pull request by swang392**
**Overview**: Rolling out the Dr. CI comment! Expanded the allow list to include Pytorch Dev Infra. 

**Test Plan:** all test cases are working.

### CODE REVIEW

File: torchci/lib/drciUtils.ts [hunk 1]
```diff
@@ -3,14 +3,16 @@ export const REPO: string = "pytorch";
...
 export const DOCS_URL = "https://docs-preview.pytorch.org/";
 export const PYTHON_DOCS_URL = "/index.html";
 export const CPP_DOCS_URL = "/cppdocs/index.html";
 export const DRCI_COMMENT_END = `\n
 This comment was automatically generated by Dr. CI and updates every 15 minutes.
 <!-- drci-comment-end -->`;
-export const POSSIBLE_USERS = ["swang392"];
+export const POSSIBLE_USERS = ["swang392", "janeyx99", "atalman", "clee2000", "seemethere",
+    "huydhn", "zengk95", "linbinyu", "suo", "dagitses", "malfet", "mehtanirav", "kit1980",
+    "b0noI", "weiwangmeta", "ZainRizvi"];
```
Comment (clee2000) [inline torchci/lib/drciUtils.ts:15]: i think you might be missing a few people? specifically jean, danylo, and omkar

File: torchci/lib/drciUtils.ts [hunk 2]
```diff
@@ -3,14 +3,16 @@ export const REPO: string = "pytorch";
...
-  "https://github.com/pytorch/pytorch/wiki/Dev-Infra-Office-Hours";
+    "https://github.com/pytorch/pytorch/wiki/Dev-Infra-Office-Hours";
 export const DOCS_URL = "https://docs-preview.pytorch.org/";
 export const PYTHON_DOCS_URL = "/index.html";
 export const CPP_DOCS_URL = "/cppdocs/index.html";
 export const DRCI_COMMENT_END = `\n
 This comment was automatically generated by Dr. CI and updates every 15 minutes.
 <!-- drci-comment-end -->`;
-export const POSSIBLE_USERS = ["swang392"];
+export const POSSIBLE_USERS = ["swang392", "janeyx99", "atalman", "clee2000", "seemethere",
```
Comment (zengk95) [inline torchci/lib/drciUtils.ts:13]: can we move this into rolloutUtils.ts? That we have all of our allowlist logic in the same place.

**Review by zengk95 — APPROVED** [review]
lgtm

File: torchci/lib/rolloutUtils.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export const POSSIBLE_USERS = ["swang392", "janeyx99", "atalman", "clee2000", "seemethere",
+    "huydhn", "zengk95", "linbinyu", "suo", "dagitses", "malfet", "mehtanirav", "kit1980",
```
Comment (zengk95) [inline torchci/lib/rolloutUtils.ts:2]: I don't think this is the rolloutUtils that currently exists. it's in lib/bot/rolloutUtils

Comment (swang392) [reply torchci/lib/rolloutUtils.ts:2]: oh oops

**Review by malfet — APPROVED** [review]

**Pull request merged**